Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -1391,11 +1391,11 @@ void ctkDICOMBrowser::exportSeries(QString dirPath, QStringList uids)
     // show progress
     if (d->ExportProgress == 0)
     {
-      d->ExportProgress = new QProgressDialog(this->tr("DICOM Export"), "Close", 0, 100, this, Qt::WindowTitleHint | Qt::WindowSystemMenuHint);
+      d->ExportProgress = new QProgressDialog(tr("DICOM Export"), "Close", 0, 100, this, Qt::WindowTitleHint | Qt::WindowSystemMenuHint);
       d->ExportProgress->setWindowModality(Qt::ApplicationModal);
       d->ExportProgress->setMinimumDuration(0);
     }
-    QLabel *exportLabel = new QLabel(this->tr("Exporting series ") + seriesNumber);
+    QLabel *exportLabel = new QLabel(tr("Exporting series ") + seriesNumber);
```

Comment:
```suggestion
    QLabel *exportLabel = new QLabel(tr("Exporting series %1").arg(seriesNumber));
```

Reply:
Thanks for that remark. That update is done in the upcoming commit related to string marking. This PR was just for resolving lupdate warnings.
If it's important to do it now too, I can make the related commit...

Reply:
Thanks, the update is done now.